src/framework/ui/dropdown/droppdownItem.component.tsx

```diff
@@ -31,6 +31,8 @@ type TextElement = React.ReactElement<TextProps>;
 type DefaultItemElement = React.ReactElement<TouchableOpacityProps>;
 type MultiSelectItemElement = React.ReactElement<ViewProps>;
 
+const itemTestId = (text: string): string => `@dropdown-item/${text}`;
```

Comment:
There is a better way to do this (with `getAllBy*` query). Explained above

Reply:
refactored